File: BEE 1008.cpp
#include<iostream>
#include<iomanip>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(2) << (x)
using namespace std;
int main(){
    int num, working_hour;
    float hour_rate;
    cin>>num;
    cin>>working_hour;
    cin>>hour_rate;
    cout<<"NUMBER = "<<num<<endl;
    cout<<"SALARY = U$ "<<FIXED_FLOAT(working_hour*hour_rate)<<endl;
    return 0;
}

File: BEE 1008.php
<?php

    $employeeNumber = readline();
    $workingHour = readline();
    $amountPerHour = readline();

    $salary = $workingHour * $amountPerHour;
    $salary = number_format($salary, 2, ".", "");

    echo "NUMBER = $employeeNumber\n";
    echo "SALARY = U$ $salary\n";

?>